style.css

```diff
@@ -0,0 +1,699 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  min-width: 300px;
+  max-width: 1300px;
+  color: black;
+  font-family: Arial, Helvetica, sans-serif;
+  line-height: 1em;
+  margin: 0 auto !important;
+}
+
+nav {
+  width: 100%;
+}
+
+.navigation {
+  display: grid;
+  grid-template-columns: auto;
+  color: white;
+}
+
+.card {
+  border: none;
+}
+
+.border-primary {
+  color: #7c7c7c !important;
+}
+
+.first-grid a {
+  text-decoration: none;
+  color: white;
+}
+
+.second-grid a {
+  text-decoration: none;
+  color: black;
+}
+
+.newsweek a {
+  text-decoration: none;
+  color: #f72210;
+}
+
+.world a {
+  text-decoration: none;
+  color: #f72210;
+}
+
+.world2 a {
+  text-decoration: none;
+  color: #f72210;
+}
+
```

Comment:
Let's avoid repetition of code and follow the DRY principle. you can do something like this `.world a, .world2 a, .newsweek a {
  text-decoration: none;
  color: #f72210;
}`